tls/s2n_connection.c

```diff
@@ -23,6 +23,11 @@
 #include <s2n.h>
 #include <stdbool.h>
 
+#include <openssl/crypto.h>
+#if defined(OPENSSL_IS_AWSLC)
+#include <openssl/mem.h>
+#endif
```

Comment:
(not related to your PR; just a rant) This is unfortunate. I wish we had a libcrypto abstraction layer to consolidate these differences to a single location. Just seems like an increasingly brittle design.

Reply:
Yes, this makes me very sad.